third_party/3/jwt/algorithms.pyi

```diff
@@ -0,0 +1,2 @@
+class Algorithm:
```

Comment:
Yes, exactly. Come to think of it, setting it to Any may also cause trouble because mypy has a flag to disallow inheriting from Any, so you may have to add a `# type: ignore`.